tools/metaquantome/metaquantome_viz.xml

```diff
@@ -0,0 +1,433 @@
+<tool id="metaquantome_viz" name="metaQuantome: visualize" version="@GVERSION@">
+    <description>taxonomic analysis, functional analysis, and function-taxonomy analysis results</description>
+    <macros>
+        <import>macros.xml</import>
+        <xml name="barcol">
```

Comment:
I would put those in the global macro as well I think.

Reply:
I was putting only shared (global) macros in the macros.xml. I like these here because they're only used in the viz module. But I might move them if you think they are better placed in macros.xml.

Reply:
Thats fine if you like this better. What you can also do is using a second macro file.

Reply:
yeah, i could see that the tool would be cleaner that way. i'll keep it this way for now